src/Catel.Core/Catel.Core.Shared/Threading/Extensions/ReaderWriterLockSlimExtensions.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
```

Comment:
No problem about the repetition, we can indeed take care of that later. Could you please add a file header here?: